internal/pkg/cli/pipeline_deploy.go

```diff
@@ -270,8 +274,14 @@ func (o *deployPipelineOpts) Execute() error {
 		PermissionsBoundary: o.app.PermissionsBoundary,
 	}
 
+	overrider, err := clideploy.NewOverrider(o.ws.PipelineOverridesPath(o.pipeline.Name), o.appName, "", afero.NewOsFs(), o.sessProvider)
```

Comment:
nit: maybe consider wrapping the parameters into a separate struct "NewOverriderOpts" because otherwise it makes less sense to see why we pass an empty string as a parameter.